vars/wrappedNode.groovy

```diff
@@ -9,6 +9,10 @@ def call(Map vars, Closure body=null) {
             // or is misconfigured, these operations will fail and the exception will be
             // propogated.
             withChownWorkspace { echo "cleanWorkspace: Ensuring workspace is owned by ${env.USER}" }
+            echo "Removing all docker images"
+            sh("for image in \$(docker ps -aq); do docker rm -f \$image; done")
```

Comment:
Could we add something a bit more elegant, such as:
`docker system prune --all --force` 

The above doesn't actually remove images, but rather containers that are running on the host.

Reply:
That does remove images, also not all hosts this would run on would have the `system` command. A lot of nodes are still on `1.12.6`.